projects/contributor-experience.md

```diff
@@ -0,0 +1,115 @@
+* No structured collaboration across SIGs, especially in the case of cross-cutting
+  concerns (Security, Docs, End-User, ...), leading to information silos and misalignment.
+* SIGs are short of contributors and require some help to advertise their need
+  (blog post, social posts, etc) and to lower the barrier of entry into their
+  sub-project.
+* _add more_
+
+Note that there are individuals within the project who try to tackle some of those
+aspects as individuals and not in a team effort, notable:
+
+* [Add skeleton for maintainer and contributor guidebooks](https://github.com/open-telemetry/community/pull/2051)
+* Mentorship efforts, e.g. [Outreachy](https://cloud-native.slack.com/archives/C060GFUL0P6)
+* Surveys by the End User SIG that aim towards contributor experience
+* _add more_
+
+Also related to this initiative is the [Check-In process of the Governance Committee](https://github.com/open-telemetry/community/blob/main/gc-check-ins.md)
+
+### Goals, objectives, and requirements
+
+The goal of this project is to establish a cross-cutting "SIG Contributor Experience",
+which is "responsible for improving the experience of those who upstream contribute to
+the OpenTelemetry project. [They] do this by creating, and maintaining programs and processes that promote community health and reduce project friction, while retiring those programs and processes that don't. Being conscientious of our contributor base is critical to scaling the project, growing the ecosystem, and helping the project succeed". (via [K8s Contributor Experience Special Interest Group Charter](https://github.com/kubernetes/community/blob/master/sig-contributor-experience/charter.md))
+
+While this is "yet another SIG" initiative, we believe that **not** establishing
+such a SIG is not an option: the problems outlined above will get bigger not
+smaller and while the final ownership of the health of the project is with the GC
+this requires a coordinated effort and more hands across the project to be addressed
+successfully.
+
+Note that similar to other cross-cutting SIGs (Docs, Security, End-User) this SIG will not exclusively own
+the domain of contributor experience, but they take responsibility to drive efforts in close collaboration
+with SIGs and Committees.
+
+## Deliverables
+
+After foundation the SIG will collaborate with contributors of existing efforts
+(see above) and work towards establishing programs and initiatives around contributor
+experience, including
+
+* Onboarding rituals for members, approvers, maintainers
+* A contributor guidebook
+* Mentorship opportunities
+* Contributor recognition programs (e.g. via [Credly](https://credly.com) or with blog posts on the style of "here's what's happening under the hood"
+* Blog posts and social media posts to highlight areas where help is needed
+* Events targeting the Contributor Community
+* Consume SIG Check-In reports from the GC and use this data to improve contributor experience
```

Comment:
Do we think maintaining and/or enforcing a code of conduct should be a responsibility of this SIG, or does that stay with GC?

Reply:
Does contributor diversity - both transparency reports and recruiting - feel like something that should be on this list?

Reply:
If we're able to support this, it might be nice to add a second weekly maintainer meeting that's friendly to more time zones, and to re-examine together with GC how those meetings could best serve maintainers - I observe that not all SIGs send representatives.

Reply:
> Should maintaining and/or enforcing the code of conduct be a responsibility of this SIG?

The SIG can help with that, but this is something that remains with the @open-telemetry/governance-committee as a responsibility.

> Does contributor diversity feel like something that should be on this list?

Yes! 🙌 

> If we're able to support this, it might be nice to add a second weekly maintainer meeting that's friendly to more time zones

I have thought about this a few times as well. This is now one of the examples where such a SIG can bring a proposal to another SIG (or in that case to the group of maintainers/the GC) but it's not within their capability to execute that.

I will update the list with items trying to address your suggestions